aiohttp/client.py

```diff
@@ -918,6 +919,60 @@ def timeout(self) -> Union[object, ClientTimeout]:
         """Timeout for the session."""
         return self._timeout
 
+    @property
+    def default_headers(self) -> 'CIMultiDict[str]':
```

Comment:
```suggestion
    def headers(self) -> 'CIMultiDict[str]':
```
Let's keep property names in sync with constructor parameters